client/src/ingredients/browser-compatibility-table/error-boundary.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+/**
+ * The error boundary for BrowserCompatibilityTable.
+ *
+ * When the whole BrowserCompatibilityTable crashes, for whatever reason,
+ * this component will show a friendly message
+ * to replace that crashed component
+ */
+export class BctErrorBoundary extends React.Component {

```

Comment:
```suggestion
export class BrowserCompatibilityErrorBoundary extends React.Component {
```
is more in line with the naming used in `index.js`.

Reply:
Thanks for the suggestion!